src/components/navigation/NavigationMenu.vue

```diff
@@ -19,11 +19,18 @@
             <aside class="menu">
                 <p class="menu-label">{{ activeGame.displayName }}</p>
                 <ul class="menu-list">
-                    <li><a href="#" @click="launchModded"><i class="fas fa-play-circle icon--margin-right"/>Start modded</a>
+                    <li v-if="canShowServerList">
+                        <a href="#" data-ref="serverlist" @click="emitClick($event.target)"
+                           class="tagged-link" :class="[view === 'serverlist' ? 'is-active' : '']">
+                            <i class="fas fa-server tagged-link__icon icon--margin-right" data-ref="serverlist" @click.prevent.stop="emitClick($event.target)"/>
+                            <span class="tagged-link__content margin-right margin-right--half-width" data-ref="serverlist" @click.prevent.stop="emitClick($event.target)">Server List</span>
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" @click="launch(true)"><i class="fas fa-play-circle icon--margin-right"/>Start modded</a>
```

Comment:
We should be avoiding passing what's essentially a magic value where possible. A better approach would be to either pass an enum (EG: LaunchType.MODDED), or to keep the multiple methods separation. 

Removing the method separation should be alright now because it of the argument resolvers. Originally they behaved differently.